features/steps/steps.py

```diff
@@ -352,6 +354,21 @@ def then_stdout_matches_regexp(context):
     assert_that(context.process.stdout.strip(), matches_regexp(context.text))
 
 
+@then("stdout is formatted as `{output_format}` and has keys")
+def then_stdout_is_formatted_and_has_keys(context, output_format):
+    output = context.process.stdout.strip()
+    if output_format == "json":
```

Comment:
should we have specific errors overall?
Here for instance, should the test inform that only `json` or `yaml` are valid formats given it comes from the test scenario itself?

right now, it will fail because `data` does not exist

Reply:
No, I don't think we need to have anything like this at the moment. The test should fail loudly if we add a new format without changing the code, so we can spot that and fix it